changes/4961.changed

```diff
@@ -0,0 +1,2 @@
+Widened left navigation menu and made longer menus truncate instead of overflowing into the next line.
```

Comment:
Is this for menu items too or only for top-level menu tabs? (E.g. does "Interface Redundancy Groups" still get wrapped, or is it now truncated?)

Reply:
This is only for top level items:

![image](https://github.com/nautobot/nautobot/assets/75227981/324e7f95-c3bb-4478-8702-44970a3c4361)

This is what it would look like if applied to all items:

![image](https://github.com/nautobot/nautobot/assets/75227981/6e94a55c-d2b4-487c-9bd7-162b64ff8edf)

Here's what device lifecycle would look like, notice the 4 reports at the bottom:

![image](https://github.com/nautobot/nautobot/assets/75227981/ac51a82b-d18b-4464-9055-c7f9e05c1581)